terraform/services/ecs_service/alb.tf

```diff
@@ -12,7 +12,8 @@ resource "aws_alb_target_group" "ecs_service" {
   }
 }
 
-resource "aws_alb_listener_rule" "rule" {
+resource "aws_alb_listener_rule" "path_rule" {
+  count        = "${1 - var.use_host_condition}"
```

Comment:
I think we can use [conditionals](https://www.terraform.io/docs/configuration/interpolation.html#conditionals) to achieve the same effect, and do away with the need for a `host_condition` variable. Something like:

```hcl
count  = "${var.host_name == "" ? 0 : 1}"
```